.github/workflows/argilla-frontend.workflow.yml

```diff
@@ -0,0 +1,87 @@
+name: Build Argilla frontend package
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+on:
+
+  workflow_call:
+
+  pull_request:
+    paths:
+      - "argilla-frontend/**"
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+
+  build:
+    name: Build argilla-frontend
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Code 🛎
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
Are we not using version 18?

Reply:
Not sure which version we're using. In any case, it would be nice to keep updated as much as possible. cc @damianpumar